schnorr_fun/src/frost.rs

```diff
@@ -60,8 +59,9 @@
 //! let message =  Message::plain("my-app", b"chancellor on brink of second bailout for banks");
 //! // Generate nonces for this signing session.
 //! let session_id = b"signing-cool-message-attempt-1".as_slice(); // ⚠ must be different for every session
-//! let my_nonce = proto.gen_nonce(&my_secret_share, session_id, Some(frost_key.public_key()), Some(message));
-//! # let nonce3 = proto.gen_nonce(&secret_share3, session_id, Some(frost_key.public_key()), Some(message));
+//! let mut nonce_rng: ChaCha20Rng = proto.gen_nonce_rng(&my_secret_share, session_id, Some(frost_key.public_key()), Some(message));
+//! let my_nonce = proto.gen_nonce(&mut nonce_rng);
+//! # let nonce3 = proto.gen_nonce(&mut nonce_rng);
```

Comment:
probably a bad idea to get nonce randomness from another notional party even in hidden doc example. You can just let `nonce3 = binonce::NonceKeyPair::random(&mut rand::thread_rng())`.